app/services/IdentityService.scala

```diff
@@ -49,6 +52,23 @@ class IdentityService(apiUrl: String, apiClientToken: String)(implicit wsClient:
 
   import IdentityServiceEnrichers._
 
+  private def request(path: String): WSRequest = {
```

Comment:
You should be able to use the existing get() method here I think rather than adding a new method

Reply:
But I'm making a post request, not a get request